src/event.rs

```diff
@@ -372,6 +372,16 @@ pub enum WindowEvent<'a> {
     ///
     /// At the moment this is only supported on Windows.
     ThemeChanged(Theme),
+
+    /// The window has been occluded (completely hidden from view).
+    ///
+    /// This is different to window visibility as it depends on whether the window is closed,
+    /// minimised, set invisible, or fully occluded by another window.
+    ///
+    /// ## Platform-specific
+    ///
+    /// At the moment this is only supported on macOS and X11.
```

Comment:
Could you describe platform section in the common format we use? like

```
/// Platform-specific behavior:
/// - **iOS / Android / Web :** Unsupported.
```

Reply:
I added wayland into that as well